Source/JavaScriptCore/inspector/protocol/CSS.json

```diff
@@ -402,6 +404,18 @@
                 { "name": "rule", "$ref": "CSSRule", "description": "The resulting rule after the selector modification." }
             ]
         },
+        {
+            "name": "setGroupingHeaderText",
+            "description": "Modifies an @rule grouping's header text.",
+            "targetTypes": ["page"],
+            "parameters": [
+                { "name": "ruleId", "$ref": "CSSRuleId" },
```

Comment:
This works in the commits from today - The RuleId is a specific enough identifier in this case, and we now share CSSGrouping model objects within a DOMNodeStyle object since they can be displayed together. This means that the single representation of the group for the current DOMNodeStyle will be updated, and will in turn notify its clients of the update, resulting in the text being updated for other occurrences of the same rule. When switching to another node to view its styles, this update is automatic because a style sheet change will have occurred, marking the DOMNodeStyle as dirty, and it will be refreshed before display.

Reply:
nice!  do we have tests for this exact scenario? 😃